task/jq/0.1/README.md

```diff
@@ -0,0 +1,31 @@
+# JQ Task
+
+The following task is used to execute a given JQ script and expose its result. It is able to work with both JSON string and file as input to the script.
+
+## Installing the Task
+
+```bash
+kubectl apply -f https://api.hub.tekton.dev/v1/resource/tekton/task/jq/0.1/raw
+```
+
+## Parameters
+
+- **options**: JQ options that will be used. If none is provided, the default value "" will be used."
+- **filter**: JQ filter that will be executed. If none is provided, the the filter '.' will be executed by default. The filter does not need to be wrapped with ''.
+- **image**: The jq image to use.
+- **input**: The input to be passed to the JQ command. It can be a JSON string or a file name. In case of file pass the entire path where the file is mounted.
+- **stringOrFile**: Flag used to determine whether the input passed will be a JSON string or file. Use **string** or **file**.
+
+## Workspaces
+
+- **source** : The workspace containing files on which we want to handle. This workspace can be a persistent volume or configmap.
+
+## Platforms
+
+The Task can be run on `linux/amd64` platform.
+
+## Usage
+
+For an example of real-world usage, check the **tests** directory for this task for an example of how it can be used.
```

Comment:
should we add a hyperlink to tests ? someone viewing from hub.tekton.dev, it will be easy for them to navigate. WDYT?

Reply:
That's an excellent idea.